core/src/avm2/globals/q_name.rs

```diff
@@ -92,7 +92,8 @@ pub fn init<'gc>(
     };
 
     if let Some(namespace) = namespace {
-        this.set_namespace(activation.context.gc_context, namespace)
+        this.set_namespace(activation.context.gc_context, namespace);
+        this.set_is_qname(activation.context.gc_context, true);
```

Comment:
Why is this conditional on the namespace?

Reply:
@adrian17 I think you answered this on Discord, can you put the comment here so we can keep track of it?

Reply:
TBH I was waiting for some ack or response there, as the string->xml-name stuff distinction between avmplus and us really confused me :/
I'll get back to it after the '+' parsing regression.